Xcode/xcconfigs/Project.xcconfig

```diff
@@ -31,10 +31,10 @@ ONLY_ACTIVE_ARCH                                   = NO
 SKIP_INSTALL                                       = YES
 SUPPORTED_PLATFORMS                                = macosx iphoneos iphonesimulator appletvos appletvsimulator
 
-VALID_ARCHS                                        = x86_64 arm64 armv7 armv7s i386
-VALID_ARCHS[sdk=macosx*]                           = x86_64
-VALID_ARCHS[sdk=iphonesimulator*]                  = x86_64                    i386
-VALID_ARCHS[sdk=appletvsimulator*]                 = x86_64                    i386
+VALID_ARCHS                                        = x86_64 arm64 armv7 armv7s
+VALID_ARCHS[sdk=macosx*]                           = x86_64 arm64
+VALID_ARCHS[sdk=iphonesimulator*]                  = x86_64
```

Comment:
officially we still support 32 bit iPhone simulator architecture right?

Reply:
I was assuming that a 32-bit simulator build was needed only for 32-bit Macs, but I think that's wrong; apparently it's needed to run a 32-bit iOS app in the simulator.

macOS Catalina can't run 32-bit code anymore, so you'd have to be on 10.14 or earlier, and you'd have to install iOS 10 into the Simulator because that's the last that can run 32-bit, but it's doable. So I'll put it back for now.

But I'm really hoping that for 2.0/Lithium we can dump 32-bit and go to iOS 11+.